src/datumaro/experimental/dataset.py

```diff
@@ -507,6 +507,61 @@ def append_dataset(self, dataset: Dataset) -> None:
         converted_dataset = dataset.convert_to_schema(target_dtype_or_schema=self.schema)
         self.df = self.df.vstack(converted_dataset.df)
 
+    def export(
+        self,
+        output_path: Union[str, Any],
+        export_images: bool = True,
+        as_zip: bool = False,
+    ) -> None:
+        """
+        Export this dataset to disk in a structured format.
+
+        The dataset is exported with the following structure:
+        - data.parquet: The DataFrame in Parquet format
+        - metadata.json: Schema, categories, and image paths
+        - images/: Directory containing exported images (if export_images=True)
+
+        Image format is automatically determined:
+        - ImagePathField: Preserves original format (copied directly)
+        - ImageCallableField: Saved as PNG (lossless)
+        - InstanceMaskCallableField: Saved as PNG (best for masks)
+
+        Args:
+            output_path: Path to export to (directory or .zip file)
+            export_images: Whether to export images from callable/path fields
+            as_zip: Whether to package everything as a ZIP file
+
+        Raises:
+            ValueError: If the dataset has transforms (not yet supported)
+        """
+        from .export_import import export_dataset
```

Comment:
Does this import have to be in the function, not at the top? To be honest, I think we do not need to define class methods for export and import. These are huge operations and I believe it makes more sense to keep these functions separate from the Dataset class.

Reply:
There are import loops if put at the top. I agree with you that this doesn't necessarily need to be part of the class. I will move it out of here.